src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -41,12 +56,16 @@ async function boot() {
   circuitHandlers.forEach(h => h.onConnectionUp && h.onConnectionUp());
 }
 
-async function initializeConnection(circuitHandlers: CircuitHandler[]): Promise<signalR.HubConnection> {
-  const connection = new signalR.HubConnectionBuilder()
+async function initializeConnection(configureSignalR: SignalRBuilder | null, circuitHandlers: CircuitHandler[]): Promise<signalR.HubConnection> {
```

Comment:
null is not cool in the JS world. May I suggest you use SignalRBuilder | undefined? Or better yet. The more idiomatic js way of doing this is through the destructuring syntax `({ configureSignalr: SignalRBuilder?, circuitHandlers: CircuitHandler[]})` (You can do this on JS parameter lists)

Reply:
Does this matter though? How would an undefined get in here? This isn't something users call directly.

Reply:
I might have been throw off by the typing. You are right it doesn't matter, it's just the typing is not idiomatic.